designs/virtualization/4.6/Upload-image-prior-to-creating-VM/Upload Image prior to creating vm.md

```diff
@@ -0,0 +1,37 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Upload Image prior to creating a VM
+
+This PR deals with when a user wants to upload an image prior to creating a virtual machine.
+It needs to be accomplished via the storage/PVC area.
+It should ensure the user understands what the backing PVC does but does not remove the Disk terminology.
+
+The ‘Create PVC’ button will change into a dropdown, so users will be able to select ‘Upload data to Persistent Volume Claim’ from this dropdown. 
+
+![PVC list with drop down button](img/PVC-dropdown.png)
```

Comment:
It makes me wonder if this selection should be on the Create form. You are still creating a PVC via a form...maybe there should be an option in the form to upload data which exposes a "Local File Path" field?

Reply:
Ah yes sorry, I didn't explain my thought fully :) 

In this design, we are suggesting that there are separate actions for creation, one of them being to "Upload data to Persistent Volume Claim". My thought is that if we just allow it to be one single "New with form" selection then the user could choose in the form that they want to upload data to the PVC. I'm just thinking it would fit a bit more naturally to choose to upload data or not in the creation form rather than choose from the creation dropdown.

Did I make any more sense this time? Happy to chat about it further over a call if needed!

Reply:
@lizsurette We discussed this early on with the dev team and they opted for creating a new object rather than adjusting the current create PVC.
I believe part of it was to not hide a new feature in an existing form but highlight as a new action that can be taken. 
It stems from CNV features being added in the UI and how to ensure users find them. Being able to upload is a really nice addition so we are hoping this helps users discover it. 
If we want to change the design I'd want to discuss with the dev side to hear their perspective as well.

Reply:
Ah I understand that for technical reasons and for easier extensibility :) Thanks for explaining!!

Reply:
ya good point though about documenting WHY we do this. @yfrimanm it's a good idea to adjust the explanation here to clarify why we are creating a dropdown and adding a new action.

Reply:
Thanks, I see your point here @lizsurette and thanks for the explanation @matthewcarleton. 
I'll add it to the documentation.